### PR TITLE
pkg/oci: Clarify resource calculation comment

### PR DESCRIPTION
**Pull request by grahamwhaley**
We use a couple of relativley simple formulas to work out how much
RAM and how many vCPUs to allocate to the VM based on the
Resource request we are passed in.
Improve the comment on these calculations to make it clear which
way they round and if they impose a floor to the resulting value.

Fixes: #554

Signed-off-by: Graham Whaley <graham.whaley@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
Just fyi, even though this is a benign comment change, I wanted to build and test check it, but don't seem able to do that on my machine right now.

File: pkg/oci/utils.go [hunk 1]
```diff
@@ -430,7 +431,12 @@ func vmConfig(ocispec CompatOCISpec, config RuntimeConfig) (vc.Resources, error)
 			return vc.Resources{}, fmt.Errorf("Invalid OCI cpu period %d", period)
 		}
 
-		// round up to 1 CPU
+		// Use some math magic to round up to the nearest whole vCPU
+		// (that is, a partial part of a quoata request ends up assigning
```
Comment (jodh-intel) [inline pkg/oci/utils.go]: Nit: "quota".

**Comment by jodh-intel** [comment]
+1 for good comments!

lgtm

**Comment by devimc** [comment]
lgtm

**Comment by devimc** [comment]
thanks @grahamwhaley for improve the comments

File: pkg/oci/utils.go (hunk 1 above)
Comment (grahamwhaley) [reply pkg/oci/utils.go]: fixed :-)

**Comment by jodh-intel** [comment]
The CI builds all finished successfully and coveralls also ran successfully, so why it's complaining I'm not sure. Merging...

**Pull request merged**